packages/react/etc/react.api.md

```diff
@@ -1204,7 +1204,7 @@ export const getNextResizeGroupStateProvider: (measurementCache?: {
 export function getOppositeEdge(edge: RectangleEdge): RectangleEdge;
 
 // @public
-export function getPersonaInitialsColor(props: Pick<IPersonaProps, 'primaryText' | 'text' | 'initialsColor'>): string;
+export function getPersonaInitialsColor(text: string | undefined, initialsColor?: PersonaInitialsColor | string, primaryText?: string): string;
```

Comment:
another reason to revert this change is that `getPersonaInitialsColor` is public api. so changing it would be a breaking change which we cannot make